guides/common/modules/proc_running-salt.adoc

```diff
@@ -1,43 +1,40 @@
 [id="Running_Salt_{context}"]
 = Running Salt
 
-You can run arbitrary Salt functions using remote execution.
-
-Most commonly, you can run `salt.highstate` on one or more Salt Minions.
-This applies all relevant Salt States on your managed hosts.
-
-.Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Select one or multiple hosts.
-. Click *Run Salt* from the _actions_ drop down menu.
+You can run arbitrary Salt functions using remote execution, for example `salt.highstate`, on one or more Salt Minions.
+This applies all relevant Salt States on your managed hosts and includes an optional test mode that tells you what would happen without actually changing anything.
+
+.Running Salt state.highstate
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run job*.
+. Select *Salt* as *Job category* and *Salt Run state.highstate {endash} Salt default* as *Job template* and click *Next*.
+. Select hosts on which you want to run the job.
+If you do not select any hosts, the job will run on all hosts you can see in the current context.
+. Click *Next*.
+. Optional: To configure advanced settings for the job, fill in the *Advanced fields*.
+To learn more about advanced settings, see {ManagingHostsDocURL}advanced-settings-in-the-job-wizard_managing-hosts[Advanced Settings in the Job Wizard] in _{ManagingHostsDocTitle}_.
+. Click *Next*.
+. Select *Immediate execution* to execute the job immediately and click *Next*.
+. Review job details.
+You have the option to return to any part of the job wizard and edit the information.
+. Click *Run* to run `Salt highstate` and generate a Salt report.
 +
-Alternatively, you can click *Run Salt* from the _Schedule Remote Job_ drop down menu on the host overview page.
-
-Triggering a remote execution job takes you to the corresponding job overview page.
-
-If you select a host from the list, you can see the output of the remote job in real time.
-
-To view remote jobs, in the {ProjectWebUI}, navigate to *Monitor* > *Jobs*.
-
-Running `state.highstate` generates a Salt report.
-Note that reports are uploaded to {Project} every ten minutes.
+By default, reports are uploaded to {Project} every ten minutes.
```

Comment:
Actually, this is currently not the case. But IMHO, we can leave this here for now and fix it such that the 10-min upload is enabled by default again.

Reply:
"fix it" as in "I will open a PR in foreman_salt and get it merged" or as in "This is a bug, someone should please fix it"? I'd like to keep the docs working for the current state of Foreman Salt plug-in. It's no issue at all to re-add this sentence in a couple of days/weeks after it's been fixed in foreman_salt.

I am OK either way. Maybe I should just open an issue in foreman_salt to track this?

Reply:
I've just checked it further and took a look at the code. IMHO, we should drop this line:

In the past, Salt has been delivered with the `upload-salt-reports` script + a cron job has been configured that uploads the report every 10 minutes (which is the reason for this line in the docs).
Since we move to a Reactor-based automated report upload, the cron job has already been dropped and IMHO we should drop the script soon too.

So, since we don't ship the cron job anymore, we should also remove this line from the docs - if the user didn't configure a cron job manually, there is no ten-minute report upload. Either, the Salt Reactor has been configured (which makes an on-demand report upload) or there is simply no report upload at all.

Reply:
Dropped.